### ISSUE TITLE
lm_sensors no support for 3.4.0

### ISSUE THREAD

**Issue by tobes** [ultrabug/py3status#1520]
```
$ sensors -v
sensors version 3.4.0 with libsensors version 3.4.0
$ sensors -j
sensors: invalid option -- 'j'
Try `sensors -h' for more information
```

module says I need this version or higher
```
2018-09-14 09:58:50 WARNING Exception in `lm_sensors` post_config_hook() (Exception) lm_sensors.py line 211.
2018-09-14 09:58:50 INFO Traceback
Exception: install version 3.4.0+
```

so a better error

**Comment by lasers** [comment]
`install git version` ?

**Comment by lasers** [comment]
Closed via #1533 by using `sensors -u` instead of `sensors -j`.
